spec/acceptance/message_actions_spec.rb

```diff
@@ -193,7 +211,7 @@
     )
     expect(nothing.messages.size).to eq 0
 
-    sleep(5)
+    sleep(7)
```

Comment:
See #32, the spec randomly fails on travis at 5, most likely because it just misses 5 second mark for the `run_timer`.

I didn't want to risk the change again of the travis build failing on this commit, and people would assume it's because of my changes.